packages/salesforcedx-vscode-core/src/traceflag/developerLogTraceFlag.ts

```diff
@@ -35,7 +35,7 @@ export class DeveloperLogTraceFlag {
     id: string,
     startDate: string,
     expirationDate: string,
-    debugLevelId: string
+    debugLevelId: string | null
```

Comment:
Yea, I changed it so that I could set debugLevelId to `null` in the test.

Reply:
I thought about putting the validation inside of the `setTraceFlagDebugLevelInfo()` method so that I wouldn't have to change the type def, but the logic didn't seem like it belonged in the setter.

Reply:
I was wrong about this, after taking a closer look at how we use this (`forceStartApexDebugLogging.ts`) we do need this change. Sorry, didn't see that before.

As to the validation, I agree it doesn't fit as part of `setTraceFlagDebugLevelInfo()` so this is good.